File: JJB/Programmers/231207/배열_만들기_2.cpp
#include <string>
#include <vector>

using namespace std;

vector<int> solution(int l, int r) 
{
    vector<int> answer;
    
    for(int num = l; num <= r; num++)
    {
        int temp = num;
        
        while(temp)
        {
            if(!(temp%10==0||temp%10==5))
                break;
            temp /= 10;
        }
        
        if(!temp)
            answer.push_back(num);
    }
    
    if(answer.empty())
        answer.push_back(-1);
    
    return answer;
}

File: JJB/Programmers/231207/배열_만들기_2.py
def solution(l, r):
    answer = []
    
    for num in range(l, r+1):
        num_list = list(str(num))
        if(sorted(list(set(num_list))) in [['5'],['0','5']]):
            answer.append(num)
            
    if not answer:
        answer.append(-1)
    
    return answer